README.md

```diff
@@ -167,13 +168,11 @@ Thanks goes to these wonderful people
 ([emoji key](https://github.com/kentcdodds/all-contributors#emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
-
 <!-- prettier-ignore -->
```

Comment:
I did not remove this line, this happened with pre commit hook

Reply:
That must be prettier, thanks for the clarification :)